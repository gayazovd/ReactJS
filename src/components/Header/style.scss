.wrapper{
    background-image: url(../../images/header-background.png);
    background-repeat: no-repeat;
    background-position: 0px 0px;
    background-size: cover;
}
.wrapperPosition{
    display: flex;
    flex-direction: column;
    justify-content: center;
    flex: 0 0 auto;
}