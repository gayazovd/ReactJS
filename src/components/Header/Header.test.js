import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Header from './index';

configure({ adapter: new Adapter() });


describe('Header component is render', () => {
    const props = {
        children: <div className='one'>Hello World!</div>
    };
    const header = shallow(<Header {...props} />);

    it('Header snapshot', () => {
        expect(header).toMatchSnapshot();
    });

    it('Header render', () => {
        expect(header.find('.one').text()).toEqual('Hello World!')
    });
})