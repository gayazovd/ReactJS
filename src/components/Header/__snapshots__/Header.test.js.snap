// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component is render Header snapshot 1`] = `
<div
  className="wrapper wrapperPosition"
>
  <div
    className="one"
  >
    Hello World!
  </div>
</div>
`;
